lib/libc/math.c

```diff
@@ -39,17 +39,16 @@ pow(double base, double power)
 }
 
 double
-factorial(double number)
-{
-    double i;
-    double return_value = 1;
-
-    for (i = 1; i < number; i++)
-    {
-        return_value *= i;
-    }
-
-    return return_value;
+factorial(double number) {
```

Comment:
thank you very much for taking the time to contribute to the projects.

I know that I haven't written the coding style guide yet :sweat_smile:
But there is a lot of issue coding style wise

Here's a few examples of what you should do: 
- Declaring **ALL** the variable at the beginning of the function (even for the `for` statement)
- Adding a new line after each `{`, `}` and keywords (such as `if`, `else`, ...)
- Adding a new line before the `return` 
- Running `make format` before submitting your code

Reply:
Okay.